genie-web/src/main/java/com/netflix/genie/web/jpa/services/JpaServiceUtils.java

```diff
@@ -301,4 +303,17 @@ static JobMetadata toJobMetadataDto(final JobMetadataProjection jobMetadataProje
             entity.setMetadata(null);
         }
     }
+
+    static UserResourcesSummary toUserResourceSummaryDto(
+        final UserJobResourcesAggregate userJobResourcesAggregate
+    ) {
+        final String user = userJobResourcesAggregate.getUser();
```

Comment:
As an alternative you could just have the `getters` in the interface return `Optional` and use `orElse()`